File: Problem Solving/Algorithms/Warmup/Diagonal Difference.cs
using System;
using System.Collections.Generic;
using System.Linq;
class Solution {
    int diagonalDifference(List<List<int>> arr) {
        return Math.Abs(arr.Sum(x => x[arr.IndexOf(x)]) - arr.Sum(x => x[arr.Count - arr.IndexOf(x) - 1]));
    }
}

File: Practice/Problem Solving/Algorithms/Warmup/Diagonal Difference.c
int diagonalDifference(int arr_rows, int arr_columns, int** arr)
{
    int result = 0;
    
    for (int i = 0; i < arr_rows; i++)
    {
        result += arr[i][i] - arr[i][arr_rows - i - 1];
    }
    
    return abs(result);
}
